### DISCUSSION TITLE
最近版本0.3.0 0.3.1 出现音频评价报错:Error invoking remote method 'recordingsassess'

### DISCUSSION THREAD

**Discussion by anzchy** [ZuodaoTech/everyone-can-use-english#854] [Q&A]
如图，我用 enjoy 0.3.0 Macbook Pro（基于 M1 Pro芯片），录制音频后，点击评价时，出现如下错误：
Error invoking remote method 'recordingsassess':Error:unable to verify the first certificate
<img width="1512" alt="SCR-20240719-tobf" src="https://github.com/user-attachments/assets/c4b003cf-c123-44b6-ba02-5bb1fecd2545">

其次，在升级 0.3.1 后，也会出现上述问题，退出软件重新时不时出现 network error，我再三确认过电脑网络没有问题。最后不得不回退到0.3.0，但仍然没有解决上述问题。

**Comment by JIM4ME** [comment]
我也有类似的情况，也是在回退后出现。但是回退后重启软件就不会了

**Comment by asddongmen** [comment]
![image](https://github.com/user-attachments/assets/b8df348c-dfae-44c0-99fe-0ceba6d70f0f)
I also encounter this error. Is there any way to resolve it? Please help!

**Comment by anzchy** [comment]
回退到0.2.14 版本，现在报错如下：
Error invoking remote method 'recordingsassess':Error:read ECONNRESET
<img width="1512" alt="SCR-20240720-pfbr" src="https://github.com/user-attachments/assets/39a9673d-0b8e-425c-beb2-055513e13289">

**Comment by an-lee** [comment]
按这个办法试试： https://github.com/ZuodaoTech/everyone-can-use-english/discussions/853#discussioncomment-10101808
